ivy/functional/frontends/numpy/statistics/median.py

```diff
@@ -0,0 +1,23 @@
+import ivy
+
+
+def median(
```

Comment:
This is the input arguments that you are supposed to recreated I suppose: numpy.median(a, axis=None, out=None, overwrite_input=False, keepdims=False)
Also, ivy.median is not implemented yet. So you may convert your existing GitHub issue to request for the function to be added to Ivy. Meanwhile, you can select another frontend function to work on from the ToDo list!

Reply:
Or you may try using the other existing ivy functions to implement this function instead.

Reply:
@Daniel4078 where can I add the median function in ivy itself (which file)?

Reply:
Firstly, you do not need to implement ivy.median yourself as it is someone else's job. Secondly, if it is implemented, I suppose it should be in ivy\functional\ivy\statistical.py.
But as I have said, you can just change your issue into a request of implementing such function and let other professional developers deal with the problem.

Reply:
Alright, is there a function to flatten an ivy array?

Reply:
I found a flatten function in the helper functions for tests. I suppose you should not call the function, but you can use its code directly instead.
def flatten(*, ret):
    """Returns a flattened numpy version of the arrays in ret."""
    if not isinstance(ret, tuple):
        ret = (ret,)
    ret_idxs = ivy.nested_indices_where(ret, ivy.is_ivy_array)
    return ivy.multi_index_nest(ret, ret_idxs)